DigitalLearningSolutions.Web/Views/NotificationPreferences/Index.cshtml

```diff
@@ -0,0 +1,47 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.MyAccount
+
+@model NotificationPreferencesViewModel
+
+@{
+  ViewData["Title"] = "Notification Preferences";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 id="page-heading">Notification Preferences</h1>
+    @if (Model.AdminId.HasValue)
+    {
+      <details class="nhsuk-details nhsuk-expander">
+        <summary class="nhsuk-details__summary">
+          <span class="nhsuk-details__summary-text">
+            My admin account notifications
+          </span>
+        </summary>
+        <div class="nhsuk-details__text">
+        </div>
+      </details>
+    }
+
+    @if (Model.DelegateId.HasValue)
+    {
+      <details class="nhsuk-details nhsuk-expander">
+        <summary class="nhsuk-details__summary">
+          <span class="nhsuk-details__summary-text">
+            My delegate account notifications
+          </span>
+        </summary>
+        <div class="nhsuk-details__text">
+        </div>
+      </details>
+    }
+
+    <div class="nhsuk-back-link">

```

Comment:
Ah, Alex created a shared component for this fairly recently - an example of it being used is in the LearningContent controller view. :)